VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,254 @@
+# Validators FAQ
+
+## General concepts
+
+### What is a validator?
+
+The Cosmos Hub is based on Tendermint, which relies on a set of validators to secure the network. The role of validators is to run a full-node and participate in consensus by broadcasting votes which contain cryptographic signatures signed by their private key. Validators commit new blocks in the blockchain and receive rewards in exchange for their work. They must also participate in governance by voting on proposals.
+
+### What is 'staking'?
+
+The Cosmos Hub is a public Proof-Of-Stake (PoS) blockchain, meaning that validator's weight is determined by the amount of staking tokens (Atoms) bonded as collateral. These Atoms can be staked directly by the validator or delegated to them by Atom holders.
+The weight of a node determines wether or not it is a validator, and also how frequently this node will have to propose a block and how much reward it will obtain. Initially, only the top 100 validator candidates with the most weight will be validators. If validators double sign, are frequently offline or do not participate in governance, their staked Atoms (including Atoms of users that delegated to them) can be slashed.
```

Comment:
`The weight of a node` ... `node` is undefined and ambiguous. Maybe we can be more clear re "The size of a staked deposit determines" or something like this. Perhaps we want to mention the idea of candidates first, and then explain that the top 100 will be validators.

Reply:
" can be slashed" -> "can be destroyed, or 'slashed'"

Reply:
Added `Any user in the system can declare its intention to become a validator by sending a "declare-candidacy" transaction. From there, they become validator candidates.`

Second comment addressed